pkg/oci/utils.go

```diff
@@ -490,7 +492,7 @@ func PodConfig(ocispec CompatOCISpec, runtime RuntimeConfig, bundlePath, cid, co
 		return vc.PodConfig{}, err
 	}
 
-	resources, err := vmConfig(ocispec, runtime)
```

Comment:
Please let's keep two separate functions here. One related to the pod (VM) and its global resources, and the other related to each container. 

This way, no need to specify an embedded field `Resources` inside `ContainerConstraints`